tests/test_latest_pointer.py

```
@@ -0,0 +1,239 @@
+pytestmark = [pytest.mark.unit, pytest.mark.core_downloads]
+
+
+@pytest.fixture(scope="module")
+def symlinks_supported(tmp_path_factory):
+    """Check whether os.symlink works in the temp directory."""
+    tmp = tmp_path_factory.mktemp("symlink_check")
+    probe = tmp / ".probe"
+    try:
+        os.symlink(tmp.name, probe)
+        probe.unlink()
+        return True
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)